zookeeper/include/zk_nn_client.h

```diff
@@ -28,6 +28,23 @@ typedef struct
 	char group[256];
 }FileZNode;
 
+struct TargetDN
+{
+	std::string dn_id;
+	uint64_t free_bytes;	//free space on disk
+	uint32_t num_xmits;		//current number of xmits
+
+	TargetDN(std::string id, int bytes, int xmits) : dn_id(id), free_bytes(bytes), num_xmits(xmits)
+	{
+	}
+
+	bool operator<(const struct TargetDN& other) const
```

Comment:
IS this minimizing? If tdn1 < tnd2 then tdn1.xmits > tdn2.xmits, thats what we want